### PR TITLE
Added plugins required by New Voice Media and Barclays IB

### PR DESCRIPTION
**Pull request by harniman**
Please review the plugins are listed in the correct sections

### CODE REVIEW

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ See [this guide](scm-step/README.md#supporting-workflow-from-an-scm-plugin) for
 - [X] `MercurialSCM` (`mercurial`): supported as of 1.51
 - [ ] `PerforceScm` (`p4`, not the older `perforce`): [JENKINS-24206](https://issues.jenkins-ci.org/browse/JENKINS-24206)
 - [ ] `DimensionsSCM` (`dimensionsscm`): [JENKINS-26165](https://issues.jenkins-ci.org/browse/JENKINS-26165)
+- [ ] `SubversionTagging` (`svn-tag`): [JENKINS-26949](https://issues.jenkins-ci.org/browse/JENKINS-26949)
```
Comment (jglick) [inline COMPATIBILITY.md]: Actually belongs under the build steps section.

File: COMPATIBILITY.md [hunk 2]
```diff
@@ -36,6 +37,10 @@ See [this guide](basic-steps/CORE-STEPS.md#adding-support-from-plugins) for maki
 - [ ] `JaCoCoPublisher` (`jacoco`): [JENKINS-27120](https://issues.jenkins-ci.org/browse/JENKINS-27120)
 - [ ] `Publisher` (`testng`): [JENKINS-27121](https://issues.jenkins-ci.org/browse/JENKINS-27121)
 - [ ] `GroovyPostbuildRecorder` (`groovy-postbuild`): [JENKINS-26918](https://issues.jenkins-ci.org/browse/JENKINS-26918)
+- [ ] `MSbuild` (`msbuild`): [JENKINS-26948](https://issues.jenkins-ci.org/browse/JENKINS-26948)
+- [ ] `HipChat` (`hipchat`): [JENKINS-27202](https://issues.jenkins-ci.org/browse/JENKINS-27202)
+- [ ] `Log Parser` (`log-parser`): [JENKINS-27208](https://issues.jenkins-ci.org/browse/JENKINS-27208)
```
Comment (jglick) [inline COMPATIBILITY.md]: Clearly `Log Parser` is not the class name which needs fixing here. If you do not know the class name, omit it.

Comment (jglick) [inline COMPATIBILITY.md]: (And I doubt it is actually a build step, though I have not looked yet.)

File: COMPATIBILITY.md [hunk 3]
```diff
@@ -44,6 +49,7 @@ See [this guide](basic-steps/CORE-STEPS.md#adding-support-from-plugins) for maki
 - [ ] `Xvnc` (`xvnc`): [JENKINS-26477](https://issues.jenkins-ci.org/browse/JENKINS-26477)
 - [ ] `BuildUser` (`build-user-vars`): [JENKINS-26953](https://issues.jenkins-ci.org/browse/JENKINS-26953)
 - [ ] `DashboardBuilder` (`environment-dashboard`): [PR 20](https://github.com/vipinsthename/environment-dashboard/issues/20)
+- [ ] `Timestamper` (`timestamper`): [JENKINS-27207](https://issues.jenkins-ci.org/browse/JENKINS-27207)
```
Comment (jglick) [inline COMPATIBILITY.md]: Probably not fixable.

**Comment by jglick** [comment]
Most of these probably require closer review by me to see if they are really plausible and which section they belong in.

File: COMPATIBILITY.md (hunk 2 above)
Comment (harniman) [inline COMPATIBILITY.md]: ok - sorry I thought that was the plugin name. The entry class seems to be LogParserBuildStep - is that what I should put?

File: COMPATIBILITY.md (hunk 1 above)
Comment (harniman) [inline COMPATIBILITY.md]: Ok I've moved it, and based on the understanding this is now the class name not the plugin name, I've changed it to SvnTagPublisher

**Comment by jglick** [comment]
Evaluated and added as appropriate.

**Pull request closed without merging**